doc/whats_new/v0.20.rst

```diff
@@ -611,3 +614,8 @@ Changes to estimator checks
 - Add tests in :func:`estimator_checks.check_estimator` to check that an
   estimator can handle read-only memmap input data. :issue:`10663` by
   :user:`Arthur Mensch <arthurmensch>` and :user:`Loïc Estève <lesteve>`.
+
+- :func:`estimator_checks.check_sample_weights_pandas_series` now uses 8 rather
```

Comment:
not sure we usually document utils but I guess there's no harm in documenting this.

Reply:
I agree this is not a very interesting change, but given that there was a dedicated section for changes to estimator checks already I added it